admin/app/components/users/user-detail-personal-information/authentication-method.hbs

```diff
@@ -1,95 +1,122 @@
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "EMAIL"}}
-      data-test-remove-email
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
-<div class="user-authentication-method__item" data-test-username>
-  <div class="user-authentication-method-item__label">
-    <span>Identifiant</span>
-    {{#if @user.hasUsernameAuthenticationMethod}}
-      <FaIcon @icon="check-circle" class="user-authentication-method-item__check" />
-    {{else}}
-      <FaIcon
-        @icon="times-circle"
-        aria-label="L'utilisateur n'a pas de méthode de connexion avec identifiant"
-        class="user-authentication-method-item__uncheck"
-      />
-    {{/if}}
-  </div>
-  {{#if (and @user.hasUsernameAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "USERNAME"}}
-      data-test-remove-username
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
-<div class="user-authentication-method__item" data-test-pole-emploi>
-  <div class="user-authentication-method-item__label">
-    <span>Pôle Emploi</span>
-    {{#if @user.hasPoleEmploiAuthenticationMethod}}
-      <FaIcon @icon="check-circle" class="user-authentication-method-item__check" />
-    {{else}}
-      <FaIcon
-        @icon="times-circle"
-        aria-label="L'utilisateur n'a pas de méthode de connexion avec Pôle Emploi"
-        class="user-authentication-method-item__uncheck"
-      />
-    {{/if}}
-  </div>
-  {{#if (and @user.hasPoleEmploiAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "POLE_EMPLOI"}}
-      data-test-remove-pole-emploi
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
-<div class="user-authentication-method__item" data-test-mediacentre>
-  <div class="user-authentication-method-item__label">
-    <span>Médiacentre</span>
-    {{#if @user.hasGARAuthenticationMethod}}
-      <FaIcon @icon="check-circle" class="user-authentication-method-item__check" />
-    {{else}}
-      <FaIcon
-        @icon="times-circle"
-        aria-label="L'utilisateur n'a pas de méthode de connexion avec GAR"
-        class="user-authentication-method-item__uncheck"
-      />
-    {{/if}}
-  </div>
-  {{#if (and @user.hasGARAuthenticationMethod (not @user.hasOnlyOneAuthenticationMethod))}}
-    <PixButton
-      class="user-authentication-method__remove-button"
-      @triggerAction={{fn @toggleDisplayRemoveAuthenticationMethodModal "GAR"}}
-      data-test-remove-mediacentre
-    >
-      SUPPRIMER
-    </PixButton>
-  {{/if}}
-</div>
\ No newline at end of file
+
+<table>
```

Comment:
**Accessibilité :** 
Il va s'agir ici d'une "tableau de mise en page" (layout table), Wave nous l'indique comme un warning.
Visiblement on ne doit utiliser les tableaux uniquement pour les données, pas pour un besoin de mise en page.

<img width="734" alt="Capture d’écran 2022-01-03 à 11 48 41" src="https://user-images.githubusercontent.com/58915422/147923251-6b8b3883-0cd2-4830-91c9-836e0deb128a.png">


"_The original intended use of HTML tables was for tabular data. A table is a data table when row headers, column headers, or both are present. Layout tables are also not supposed to be used in HTML5._" https://webaim.org/techniques/tables/

<img width="734" alt="Capture d’écran 2022-01-03 à 11 59 19" src="https://user-images.githubusercontent.com/58915422/147923267-c1324fc3-6762-498e-ac41-091b4a1b9855.png">

Reply:
Du coup si c'était uniquement un besoin de mise en page, on peut utiliser du css, sauf s'il y a un autre besoin que j'ai pas vu :)

Reply:
vu ensemble, revoir la structure de ce tableau pour au moins éviter qu'il soit considéré comme un layout table